### DISCUSSION TITLE
Welcome to pddp Discussions!

### DISCUSSION THREAD

**Discussion by umlaeute** [pure-data/pddp#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by porres** [comment]
Hi, I'm Alex, been biten by the Pd community in 2007 and became a Pd zombie. I've written a few tutorials on Pd and take care of 2 big libraries. I like writting tutorials and help files. I'm devoting a good amount of time to revise and rewrite Pd's help files, manual and stuff.

cheers

**Comment by abreubacelar** [comment]
Hello, i'm José de Abreu, Started to learn pd in 2014 i think (i'm still a baby here, need to learn many things)... I'm a pd enthusiast lurking in the shadows of telegram, facebook and discord groups, and like to learn pd helping new people with their projects.  

I am very interested in learning more and help with the documentation, so it is why i'm here. My current project is a parallel wiki where I am training how to make pages and search for info, https://pdwiki.miraheze.org/wiki/Main_Page , the idea started in the discord community as a wish for updated tutorials for pd, it is still in sketch stage but i hope that this may became useful in the future.

**Comment by porres** [comment]
you can use this project to share ideas and discuss the scope and goals of this wiki.
